### PR TITLE
Add profile information and image to portfolio page. 

### PR DESCRIPTION
**Pull request by keelywan**

### CODE REVIEW

**Review by nickKelman — APPROVED** [review]

File: portfolio/src/main/webapp/index.html
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:12]: This block of code and the code contained within should use 2 spaces for indentation instead of 4.

File: portfolio/src/main/webapp/index.html
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:17]: /outisde/outside

File: portfolio/src/main/webapp/index.html
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:18]: My favorite book series is the Kingkiller chronicle. I recommend trying 'The name of the wind' if you haven't already.

File: portfolio/src/main/webapp/index.html
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:16]: Indentation in this paragraph block looks a little inconsistent.

**Review by nickKelman — COMMENTED** [review]
Before submitting this change, wait for both Colette and my own approval. It's important we both get to review the code changes you make.

File: portfolio/src/main/webapp/index.html
Comment (keelywan) [reply portfolio/src/main/webapp/index.html:18]: Cool, I'll definitely check that out!

File: portfolio/src/main/webapp/index.html
Comment (keelywan) [reply portfolio/src/main/webapp/index.html:12]: Got it, thanks! I have a later commit that changes the indentation.

File: portfolio/src/main/webapp/index.html
Comment (keelywan) [reply portfolio/src/main/webapp/index.html:16]: Should I be putting the img tag on a new line?

Comment (nickKelman) [reply portfolio/src/main/webapp/index.html:16]: I think it would help with readability, seems like a good idea.

**Pull request merged**
